docs/source/getting_started/advanced.rst

```diff
@@ -116,17 +123,39 @@ learning:
 
   - Check the documentation: :py:class:`lightly.models.simsiam.SimSiam`
 
+- `Barlow Twins: Self-Supervised Learning via Redundancy Reduction, S. Deny, 2021 <https://arxiv.org/abs/2103.03230v1>`_
+
+  - Check the documentation: :py:class:`lightly.models.barlowtwins.BarlowTwins`
+
 Do you know a model that should be on this list? Please add an issue on GitHub :)
 
 All models have a backbone component. This could be a ResNet.
 When creating a self-supervised learning model you pass it a backbone. You need
 to make sure the backbone output dimension matches the `num_ftrs` parameter 
 of the model.
 
+Lightly has a built-in generator for ResNets. However, the model architecture slightly differs from the official ResNet implementatation.
+The difference is in the first few layers. Whereas the official ResNet starts 
+with a 7x7 convolution the one from lightly has a 3x3 convolution. 
+
+* The 3x3 convolution variant is more efficient (less parameters and faster 
```

Comment:
Is 3x3 really more efficient? I thought it was less efficient for larger images due much more features. Please clarify here.

Reply:
A 3x3 kernel has fewer parameters and is faster to compute than a 7x7 kernel. I'm referring here to just the first layer.